src/Controller/CocktailController.php

```diff
@@ -54,7 +68,26 @@ public function submitName(Request $request): Response
 
         return $this->render('cocktail/submit.html.twig', [
             'cocktailForm' => $form->createView(),
-            'result' => $cocktailName->getCocktailName()
+            'result' => $this->cocktailNameRequest($cocktailName)
         ]);
     }
+
+    private function cocktailNameRequest(CocktailName $cocktailName): string
+    {
+        $response = $this->httpClient->request('GET', self::URL, [
+            'query' => [
+                's' => $cocktailName->getCocktailName()
+            ]
+        ]);
+
+        $parsedResponse = $response->toArray();
+
+        if(empty($parsedResponse['drinks'])){
+            return 'cocktail not found';
+        }
+        else{
+            return $parsedResponse['drinks'][1]['strDrink'];
```

Comment:
Why are you returning the second element? Counting starts from 0... Also, is 'drinks' parameter non-existent when no drinks are returned? Or this parameter exists, just without any results inside. Check this and fix accordingly if needed.

Reply:
returned second element for testing purposes to see different margarita :D The parameter 'drinks' exists, containing null.

Reply:
So test locally, this code goes to main branch :) 
Anyway, your code would break here if only one cocktail would be returned. 
```suggestion
            if(empty($parsedResponse['drinks'][1])) {
                /err
            }
```